### PR TITLE
DTSPB-2729  fix Send Documents Item at Written Wishes

### PR DESCRIPTION
**Pull request by MarcusHMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-2729

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/unit/services/testCoverSheetPdf.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ describe('CoverSheetPdfService', () => {
                 will: {
```
Comment (sanjayparekh-hmcts) [inline test/unit/services/testCoverSheetPdf.js:140]: Are there any component tests that need to be changed/added?

**Review by sanjayparekh-hmcts — COMMENTED** [review]
Are therere

File: test/unit/services/testCoverSheetPdf.js (hunk 1 above)
Comment (MarcusHMCTS) [reply test/unit/services/testCoverSheetPdf.js:140]: i've not update any component and just use the component, so no need to add any new test for the component

Comment (sanjayparekh-hmcts) [reply test/unit/services/testCoverSheetPdf.js:140]: ok

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request merged**